### ISSUE TITLE
Getting "TypeError: Cannot read property 'call' of undefined" when trying to print to console

### ISSUE THREAD

**Issue by UzEE** [nickuraltsev/composite-error#1]
I'm getting `TypeError: Cannot read property 'call' of undefined` when trying to use this module and printing to console. For example, following the example given in the module:

```js
const firstError = new Error('First error');
const secondError = new CompositeError('Second error', firstError);
const thirdError = new CompositeError('Third error', secondError);

console.log(thirdError.stack);
```

Prints the following:

```
TypeError: Cannot read property 'call' of undefined
    at CompositeError.get (C:\dev\csp2\frontend\src\csp2\commands\node_modules\composite-error\lib\compositeError.js:44:44)
    at CompositeError.formatStackTraces (C:\dev\csp2\frontend\src\csp2\commands\node_modules\composite-error\lib\compositeError.js:55:20)
    at CompositeError.get (C:\dev\csp2\frontend\src\csp2\commands\node_modules\composite-error\lib\compositeError.js:24:35)
    at repl:1:23
    at ContextifyScript.Script.runInThisContext (vm.js:23:33)
    at REPLServer.defaultEval (repl.js:340:29)
    at bound (domain.js:280:14)
    at REPLServer.runBound [as eval] (domain.js:293:12)
    at REPLServer.onLine (repl.js:537:10)
    at emitOne (events.js:101:20)
```

My environment:

```
OS: Windows 10 Pro 64-bit 14393.953
Node: v7.8.0
NPM: v4.2.0
```

**Comment by nickuraltsev** [comment]
Thank you for filing this! The module expects `Error.stack` to be an accessor property, but it is a data property in node v7.

**Comment by nickuraltsev** [comment]
Fixed in v1.0.2.
